examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<p>Text comes before the second way is shown, but says there are already two ways...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>